cms/static/js/views/list.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * A generic list view class.
+ *
+ * Expects the following properties to be overriden:
+ * - emptyTemplateName (string): Name of an underscore template to
+ *   render when the collection is empty.
+ * - createItemView (function): Create and return an item view for a
+ *   model in the collection.
+ * - newModelOptions (object): Options to pass to models which are
+ *   added to the collection.
+ * - itemCategoryDisplayName (string): Display name for the category
+ *   of items this list contains.  For example, 'Group Configuration'.
+ *   Note that it must be translated.
+ */
+define([
+    'js/views/baseview'
+], function(BaseView) {
+    'use strict';
+    var List = BaseView.extend({
+        events: {
+            'click .action-add': 'addOne',
+            'click .new-button': 'addOne'
+        },
+
+        initialize: function() {
+            this.emptyTemplate = this.loadTemplate(this.emptyTemplateName);
+            this.listenTo(this.collection, 'add', this.addNewItemView);
+            this.listenTo(this.collection, 'remove', this.handleDestory);
```

Comment:
Typo in the name. Also, it seems confusing to me because it isn't handling destroy of the view. I'd call it ```onRemoveItem```, and rename the add to ``onAddItem```.